src/constants.ts

```diff
@@ -17,13 +20,65 @@ export const REPORT_ERROR_FORM_ENTRIES = {
 export const REPORT_ERROR_FORM_URL =
   "https://docs.google.com/forms/d/1YOvhK2RfTQmYO8DGMRqN7FYxRhBZd9jB6PZ7InJirTk/viewform";
 
-export const USDT_DECIMALS = 6;
-
 export const UNISWAP_V3_POOL_FEE = 3000;
 
 export const FIAT_EXCHANGE_RATES_API_URL = "https://api.exchangeratesapi.io/v1/latest";
 
-export const chains: Chain[] = [
-  { name: "Ethereum chain", icon: EthChainIcon, chainId: 1 },
-  { name: "Polygon Hermez chain", icon: PolygonHermezChainIcon, chainId: 10 },
-];
+export const CHAINS: Record<SupportedNetwork, Chain[]> = {
```

Comment:
Having a `Chain[]` is not forcing us to declare both chains. We should use a type-safer approach and define both chains separately.

Reply:
We shouldn't be attached to 2 chains neither, this array should be able to change dynamically. There should be at least two chains in the array, but as this a constant and I don't even expect it to change, I don't see necessary to implement a stricter approach.